### PR TITLE
man pages SAPHanaSR.py.7 SAPHanaSrMultiTarget.py.7: requirements

### PR DESCRIPTION
**Pull request by lpinne**
Hi,
just an update on requirements, based on bug #1218498. Might go into one of the next maintenance updates.
Regards,
Lars

### CODE REVIEW

File: SAPHana/man/SAPHanaSrMultiTarget.py.7 [hunk 1]
```diff
@@ -321,7 +321,7 @@ the sudo permission configuration
 the directory with HANA trace files
 .TP
 /usr/sap/$SID/HDB$nr/.crm_attribute.$SITE
-the internal cache for srHook status changes while Linux cluster is down, file is owned by ${SID}adm and must never be touched
+the internal cache for srHook status changes while Linux cluster is down, file is owned and r/w by ${sid}adm and must never be touched
```
Comment (angelabriel) [inline SAPHana/man/SAPHanaSrMultiTarget.py.7:324]: it would be more clear and consistent (see line 343 in this PR) to use 'read/write' instead of 'r/w'

**Review by angelabriel — CHANGES_REQUESTED** [review]
can you please take a look at my comments and fix at least the 'year'.
Thanks

File: SAPHana/man/SAPHanaSrMultiTarget.py.7 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .\" Version: 0.180.0
 .\"
-.TH SAPHanaSrMultiTarget.py 7 "27 Apr 2023" "" "SAPHanaSR-ScaleOut"
+.TH SAPHanaSrMultiTarget.py 7 "04 Jan 2023" "" "SAPHanaSR-ScaleOut"
```
Comment (angelabriel) [inline SAPHana/man/SAPHanaSrMultiTarget.py.7:3]: can you please adapt the year too even that traveling back in time may be exciting :smile:

File: SAPHana/man/SAPHanaSrMultiTarget.py.7 (hunk 1 above)
Comment (lpinne) [reply SAPHana/man/SAPHanaSrMultiTarget.py.7]: done

File: SAPHana/man/SAPHanaSrMultiTarget.py.7 (hunk 2 above)
Comment (lpinne) [reply SAPHana/man/SAPHanaSrMultiTarget.py.7]: done

**Review by angelabriel — APPROVED** [review]
Looks good to me, many thanks.

**Pull request merged**